### ISSUE TITLE
[2024/02/14]Machine Learning 輪講

### ISSUE THREAD

**Issue by hakubishin3** [wantedly/machine-learning-round-table#234]
## Why

Machine Learning 輪講は最新の技術や論文を追うことで、エンジニアが「技術で解決できること」のレベルをあげていくことを目的にした会です。

prev. https://github.com/wantedly/machine-learning-round-table/issues/232

## What

話したいことがある人はここにコメントしましょう！
面白いものを見つけた時点でとりあえず話すという宣言だけでもしましょう！

**Comment by KKaichi** [comment]
##[CONFIT: Improving Resume-Job Matching using Data Augmentation and Contrastive Learning](https://arxiv.org/abs/2401.16349)

###学会名
なし（だと思われる）
コロンビア大学の人とIntellipro Group Inc.所属の人が執筆。

**Comment by pythonista-blitz** [comment]
## [Enhanced Gradient Boosting for Zero-Inflated Insurance Claims and Comparative Analysis of CatBoost, XGBoost, and LightGBM](https://arxiv.org/abs/2307.07771)

### 概要
財産・傷害（P&C）保険業界がクレームの予測モデルを作成する際に、対象が過剰なゼロを持つ右に偏った分布を持つ。この問題を解決するため、従来のカウントモデルとバイナリモデルを組み合わせた「ゼロインフレーション」モデルを採用した。

### 手法
この論文では、保険請求データ（特にゼロインフレーション・テレマティクスデータを含む）を処理するためにブースティングアルゴリズムを使用し、クレームの頻度モデルを調査。具体的には、XGBoost、LightGBM、CatBoostという3つのメジャーなGBDTライブラリを評価、比較した。

### 結果
予測性能からCatBoostが自動車クレーム頻度モデルを開発する際に最適なライブラリであることが明らかになった。
さらに、新しいゼロインフレーション・ポアソン・ブースティングツリーモデル(ZIPBT)が提案され、データの特性に応じて他のモデルよりも優れた性能を示すことが分かった。

**Comment by hakubishin3** [comment]
## [Fairness in Ranking under Uncertainty](https://openreview.net/pdf?id=7wunGXQoC27)

**Comment by chimuichimu** [comment]
## [When Newer is Not Better: Does Deep Learning Really Benefit Recommendation From Implicit Feedback?](https://arxiv.org/abs/2305.01801)
Yushun Dong, Jundong Li, and Tobias Schnabel. 2023. In Proceedings of the 46th International ACM SIGIR Conference on Research and Development in Information Retrieval (Taipei, Taiwan) (SIGIR ’23).

### 概要
Implicitな嗜好データに基づく推薦タスクにおいて、深層学習モデルと従来型のモデルの性能比較を行った論文。精度（=HitRate）の観点では深層学習モデルは従来型モデルに劣る結果が得られたが、推薦アイテムの多様性など、深層学習モデルが勝る観点もあった。

### 実験設定
- モデル
    - 2つの深層学習モデル（MultiDAE, MultiVAE）と11の非深層学習モデルの、合計13のモデル
- データセット
    - 9つのpublicなデータセット
- 評価方法
    - Utility（Recall, HitRate, MeanRanks）
    - Diversity（Gini Index, Shannon Entropy）
    - Semantic Coherence Metrics（Semantic Coherence Index）
    - Subgroup-Specific Performance Evaluation
    	- アクティブなユーザ vs 非アクティブなユーザ
    	- 学習データのユーザの履歴に似たユーザ vs 似てないユーザ
    	- 人気のあるアイテム vs 人気のないアイテム

### 結果
- HitRateでは線形モデルが深層学習モデルを上回る一方、MeanRankは深層学習モデルの性能が良かった
- 2つの多様性の評価指標において、深層学習モデルが従来モデルを上回った
- Subgroup間の比較では、深層学習モデルは従来モデルに比べてより堅牢性（＝group間で精度の大きな劣化がない）があった

**Comment by nogawanogawa** [comment]
# RAPTOR: RECURSIVE ABSTRACTIVE PROCESSING FOR TREE-ORGANIZED RETRIEVAL
## 論文URL

https://arxiv.org/abs/2401.18059

## 著者

Parth Sarthi, Salman Abdullah, Aditi Tuli, Shubh Khanna, Anna Goldie, Christopher D. Manning

## 会議

ICLR 2024

## 背景

大規模な言語モデルであっても、特定のタスクに対するドメイン固有の知識が不足するが、fine tuningで世界に追従するのは難しい。
こうした状況に対して、LAGが採用されることが非常に多い。検索補強（retrieval augmentation）は有効ではあるが、既存手法は短い連続したチャンクを数個検索するのが現状である。

「シンデレラはどうやってハッピーエンドにたどり着いたのか」という質問にこうした手法では検索された数個のチャンクの情報では不十分であり、対応することができない。

## 目的

異なるレベルの質問に効果的かつ効率的に答えることができる検索手法の開発

## アプローチ

- RAPTOR
  - テキストのチャンクをクラスタ化
  - それらのクラスタのテキスト要約を生成
  - それを繰り返し、下から上にツリーを生成する

<img width="712" alt="image" src="https://github.com/nogawanogawa/paper_memo/assets/34205733/51fdf50c-c746-4842-9f39-89059853ec55">

## memo

https://github.com/nogawanogawa/paper_memo/issues/104
